sdl_android/src/main/java/com/smartdevicelink/protocol/WiProProtocol.java

```diff
@@ -753,13 +844,21 @@ public void StartProtocolService(SessionType sessionType, byte sessionID, boolea
 		handlePacketToSend(header);
 	}
 
+	@Override
+	public void RegisterSecondaryTransport(byte sessionId) {

```

Comment:
Method should be correct camel case regardless of what exists in other methods. Signature should also take in the specific transport that should be registered on.